Test/CaptureHandleReaderDeviceTest.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+using PacketDotNet;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Test
+{
+    [TestFixture]
+    public class CaptureHandleReaderDeviceTest
+    {
+        private static int capturedPackets;
+
+        private SafeHandle GetTestFileHandle(string filename)
+        {
+            filename = TestHelper.GetFile(filename);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // On Windows, we need an actual OS handle.
+                // The FileStream is not disposed, which is alright because we take care of its handle.
+                // On .NET 6+, this can be replaced with File.OpenHandle().
+                return File.Open(filename, FileMode.Open).SafeFileHandle;
+            }
+
+            // On other platforms, libpcap is not very interop-friendly and expects a FILE*.
+#if !WINDOWS
```

Comment:
Why is this macro used/needed?

Reply:
I thought so too at first, but the Windows CI failed at runtime because, due to the `Stdlib` reference, it tried to load some additional Windows-specific Mono assembly that wasn't there. It happens at class-load time, even if the code in question is not actually reachable on the platform.